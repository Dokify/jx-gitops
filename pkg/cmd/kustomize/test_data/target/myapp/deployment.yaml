# Source: myapp/templates/deployment.yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  name: myapp-myapp
  namespace: myapps
  annotations:
    wave.pusher.com/update-on-config-change: "true"
  labels:
    draft: draft-app
    chart: "myapp-1.0.5"
    newlabel: cheese
spec:
  selector:
    matchLabels:
      app: myapp-myapp
  replicas: 3
  template:
    metadata:
      labels:
        draft: draft-app
        app: myapp-myapp
    spec:
      containers:
      - name: myapp
        image: "gcr.io/myproject/myapp:1.0.5"
        imagePullPolicy: IfNotPresent
        ports:
        - containerPort: 8080
        livenessProbe:
          httpGet:
            path: /
            port: 8080
          initialDelaySeconds: 60
          periodSeconds: 10
          successThreshold: 1
          timeoutSeconds: 1
        readinessProbe:
          httpGet:
            path: /
            port: 8080
          periodSeconds: 10
          successThreshold: 1
          timeoutSeconds: 1
        resources:
          limits:
            cpu: 400m
            memory: 256Mi
          requests:
            cpu: 200m
            memory: 128Mi
      terminationGracePeriodSeconds:
