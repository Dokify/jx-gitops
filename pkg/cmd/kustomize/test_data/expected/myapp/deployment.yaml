# Source: myapp/templates/deployment.yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  name: myapp-myapp
  annotations:
    wave.pusher.com/update-on-config-change: "true"
  labels:
    newlabel: cheese
spec:
  replicas: 3
